daemon/api.go

```diff
@@ -433,6 +484,10 @@ func getSnapsInfo(c *Command, r *http.Request, user *auth.UserState) Response {
 		return InternalError("cannot find route for snaps")
 	}
 
+	if r.URL.Query().Get("select") == "refresh" {
+		return storeUpdates(c, r, user)
```

Comment:
Loose thought for the future: we may eventually have something similar to what we had before, with CandidateRevision pre-cached, and possibly pre-downloaded.